docs/src/dev_notes/standalone_ttir_run.md

```diff
@@ -0,0 +1,58 @@
+# Steps to Run TTIR MLIR Standalone (Without Forge-FFE):
```

Comment:
Can we update to something like: "How to run standalone MLIR, based on generated Forge-FE MLIR graphs"